docs/guide/tutorials/connect/connect.txt

```diff
@@ -1,3 +1,95 @@
 ==================
 Connect to MongoDB
 ==================
+
+Use the ``client.connect`` method to connect to a running MongoDB deployment.
+
+Connect to a Single MongoDB Instance
+------------------------------------
+
+To connect to a single MongoDB instance, specify the URI of the MongoDB
+instance to connect to.
+
+In the following example, the
+:manual:`URI connection string </reference/connection-string/>`
+specifies connecting to a MongoDB instance that is running on
+``localhost`` using port ``27017``. The ``myproject`` indicates the database
+to use.
+
+.. literalinclude:: /includes/basic-connection.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connect to a Replica Set
+------------------------
+
+To connect to a :manual:`replica set </core/replication-introduction/>` ,
+include a seedlist of replica set members and the name of the replica set in the
+:manual:`URI connection string </reference/connection-string/>` .
+
+In the following example, the connection string specifies two of the replica set members running on ``localhost:27017`` and ``localhost:27018`` and the name of the replica set (\ ``foo``\ ). 
+
+.. literalinclude:: /includes/connect-to-replicaset.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connect to Sharded Cluster
+--------------------------
+
+To connect to a :manual:`sharded cluster </core/sharded-cluster-components/>` , specify the ``mongos`` instance or instances in the :manual:`URI connection string </reference/connection-string/>` .
+
+In the following example, the connection string specifies the ``mongos`` instances running on ``localhost:50000`` and ``localhost:50001``.
+
+.. literalinclude:: /includes/connect-to-sharded-cluster.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connection Options
+------------------
+
+You can specify various connection settings in the :manual:`URI connection string </reference/connection-string/>` .
+
+For example, you can specify TLS/SSL and authentication setting.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+   const fs = require('fs');
+
+     // Read the certificate authority
+   const ca = [fs.readFileSync(__dirname + "/ssl/ca.pem")];
```

Comment:
Are there supposed to be array brackets around line 67 ` const ca = [fs.readFileSync(__dirname + "/ssl/ca.pem")];` or is that a typo?

Reply:
Per #2174, it is not *required*, but it is no wrong. Whether or not we want to advocate for a list here or not is entirely up to you.

Reply:
I'm going to resolve this, instead of having this prevent this PR from approval. My opinion is that we remove the array brackets to avoid confusion if it produces the same result. But I think that's something we can talk about and fix after the first iteration of these docs gets fixed.